tests/snapshots/tests.contrib.redis.test_redis_asyncio.test_basic_request.json

```diff
@@ -0,0 +1,28 @@
+[[
```

Comment:
These snapshots are missing the '_dd.p.dm` span tags introduced by this PR: https://github.com/DataDog/dd-trace-py/pull/3797. 

It might be faster to delete these snapshots and rerun the redis tests